streams/sqlite/voxel_stream_sqlite.cpp

```diff
@@ -966,6 +966,7 @@ void VoxelStreamSQLite::recycle_connection(VoxelStreamSQLiteInternal *con) {
 void VoxelStreamSQLite::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_database_path", "path"), &VoxelStreamSQLite::set_database_path);
 	ClassDB::bind_method(D_METHOD("get_database_path"), &VoxelStreamSQLite::get_database_path);
+	ClassDB::bind_method(D_METHOD("flush_cache"), &VoxelStreamSQLite::flush);
```

Comment:
What is the reason for having the exposed name be different from the internal C++ name?

Reply:
this function is overloaded in the class and that conflicts when exposing it, the compiler doesn't know which one to pick when referenced by name